### ISSUE TITLE
Famous  Matrices -- useful for teaching

### ISSUE THREAD

**Issue by alanedelman** [JuliaLang/LinearAlgebra.jl#25]
would be great to have hankel, toeplitz, vander, etc....
i'd be an advocate as well for 

``` julia
strang(n) = SymTridiagonal(2*ones(n),-ones(n-1))
```

in honor of Gil Strang

**Comment by mschauer** [comment]
A propos, `no method print_matrix(TTY,Array{Float64,2})` and no `method display(SymTridiagonal{Float64},)`.

**Comment by JeffBezanson** [comment]
In what context do you get those errors?

**Comment by mschauer** [comment]
Sorry, I was trying to display the matric, strang(5), display(strang(5)) and print(strang(5)) in a Win 64 system and neither worked, but I cannot reproduce it here under linux. Here\* `display` works, (`print` doesn't, but for a different reason:

```
julia> strang(n)=SymTridiagonal(2*ones(n),-ones(n-1))
# methods for generic function strang
strang(n) at none:1
julia> display(strang(5))
5x5 SymTridiagonal{Float64}:
diag:  2.0  2.0  2.0  2.0  2.0
 sub:  -1.0  -1.0  -1.0  -1.0
julia> print(strang(3))
ERROR: no method getindex(SymTridiagonal{Float64},Int32,Int32)
 in writedlm at datafmt.jl:208
 in print at show.jl:891
 in print at string.jl:7
```

_0.2.0-prerelease+3627 Commit 2288dc6_ i686-linux-gnu

**Comment by ViralBShah** [comment]
It may be useful to have a `TeachingMatrices` package.

**Comment by JeffBezanson** [comment]
I would call it `SpecialMatrices`.

**Comment by mschauer** [comment]
PS: The name Strang matrix - as "Strang" is German for _strand_ - certainly fits a type of tridiagonal matrix, by analogy with the providential Schwarzschild radius (german _black shield_) and Heaviside function.

**Comment by mschauer** [comment]
_Please disregard, fixed in Commit ce3a772 2013-09-16 04:04:18 UTC._

PPS, hope this is not to old: On Version 0.2.0-prerelease+3441, Commit dca8d76 2013-08-26 02:28:43 UTC, 686-w64-mingw32

```
julia> strang(n)=SymTridiagonal(2*ones(n),-ones(n-1))
# methods for generic function strang
strang(n) at none:1

julia> strang(5)
5x5 SymTridiagonal{Float64}:
diag: Error showing value of type SymTridiagonal{Float64}:
ERROR: no method display(SymTridiagonal{Float64},)
 in display at multimedia.jl:150
```
